typegate/src/runtimes/deno/worker.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright Metatype OÜ, licensed under the Elastic License 2.0.
 // SPDX-License-Identifier: Elastic-2.0
 
+/// <reference lib="deno.worker" />
```

Comment:
can you explain that addition 🙏 ?

Reply:
This is a marker for `deno_lsp` to bring the `WorkerOptions` types into the global scope in this module including and extra` Worker` ctor arguments. `deno check` seemed to be throwing otherwise.

Reply:
Have added a comment explaining this to the file.